mlserver/cli/build.py

```diff
@@ -43,7 +43,8 @@ def build_image(folder: str, dockerfile: str, image_tag: str) -> str:
         dockerfile_path = write_dockerfile(tmp_dir, dockerfile)
 
         build_cmd = f"docker build {folder} -f {dockerfile_path} -t {image_tag}"
-        build_env = {"DOCKER_BUILDKIT": "1"}
+        build_env = os.environ.copy()
+        build_env["DOCKER_BUILDKIT"] = "1"
```

Comment:
Oh wow, I'm guessing this is the fix for MacOS? That **must have not** been easy to find :grimacing: 

Thanks a lot for debugging and fixing this one! Across the team we use mainly Linux, so this type of issues often get undiscovered. Therefore, these kind of contributions are always extra appreciated :+1:

Reply:
Not too difficult to find. I think you would have rolled out 0.6.1 the next day once the rest of the community got access to the feature.